cypress/integration/data-browser/graphs.spec.js

```diff
@@ -1,171 +1,134 @@
-  ALL_GRAPHS_ENDPOINT,
-  buildURL,
-  firstGraphEndpoint,
-  devURL,
-} from "../../fixtures/mockGraphData-";
-
-import { isCI } from "../../support/helpers";
-
-const { HOST, ENVIRONMENT } = Cypress.env();
-
-let baseURLToVisit = isCI(ENVIRONMENT) ? "http://localhost:3000" : HOST;
-
-describe.skip("General Tests", () => {
-  it("URL Re-direct which contains base period selections and series", () => {
-    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`);
-    cy.get(".react-select__graph__control")
-      .click(0, 0, { force: true })
-      .find('[class*="-singleValue"]')
-      .eq(0)
-      .contains("How has the number of applications changed?")
-      .click(0, 0, { force: true });
-    cy.url().should("include", firstGraphURLQuery);
-    cy.url().should("eq", baseURLToVisit + firstGraphURL);
-  });
+import { isCI } from "../../support/helpers"
 
+const { HOST, ENVIRONMENT } = Cypress.env()
+
+let baseURLToVisit = isCI(ENVIRONMENT) ? "http://localhost:3000" : HOST
+
+describe("General Tests", () => {
   it("Checks <GraphsHeader/> component if overview props was not sent to the component", () => {
-    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`);
-    let overviewHeader = cy.get(".Graphs > :nth-child(2)");
-    if (!ALL_GRAPHS_ENDPOINT.endpoint) {
-      overviewHeader.contains(
-        "The following graphs present data for the financial institutions reporting HMDA quarterly data."
-      );
-    }
-  });
+    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`).contains(
+      "The following graphs present data for the financial institutions reporting HMDA quarterly data."
+    )
+  })
 
   it("Checks <GraphsHeader/> component if overview props was sent to the component", () => {
-    let overviewHeader = cy.get(".Graphs > :nth-child(2)");
-    if (ALL_GRAPHS_ENDPOINT.endpoint) {
-      overviewHeader.contains(
-        ALL_GRAPHS_ENDPOINT.overview +
-          "Though the graphs provide some insight into trends for these institutions, they should not be taken to represent the behavior of all mortgage lenders during the relevant period." +
-          "Use the menu below to select a graph."
-      );
-    }
-  });
-
-  it("Selects first graph from graph drop-down list and checks url contains query parameters", () => {
-    cy.get(".react-select__graph__control")
-      .click(0, 0, { force: true })
-      .find('[class*="-singleValue"]')
-      .eq(0)
-      .contains("How has the number of applications changed?")
-      .click(0, 0, { force: true });
-    cy.url().should("include", firstGraphURLQuery);
-    cy.url().should("eq", baseURLToVisit + firstGraphURL);
-  });
+    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
+    cy.get(".heading > :nth-child(2)").contains(
+      "The following graphs represent data for the 5x financial institutions reporting HMDA quarterly data in the current year. The graphs display current quarters and three years of historical data."
+    )
```

Comment:
Updated test to correctly handle two numbers.

Reply:
I don't see the update but I'll assume it's just in a `commit` that hasn't been pushed yet 🚀

Reply:
This has been updated.

New code:
```JavaScript
cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
    cy.wait(1000)
    cy.get(".heading > :nth-child(2)").contains(/[0-9]{1,2}/)
```

Reply:
@Michaeldremy I've expanded this one a bit to use a different pattern in Dev vs Prod. 
```
  it("Checks <GraphsHeader/> component if data from API succeedes then it checks if numbered financial institutions show in the header", () => {
    // In Dev only an alphanumeric approximation is provided (ex. 5x).  
    // In Prod we want to ensure that the count is numeric.
    let institutionCountRx = isProd(HOST) ? '[0-9]{1,3}' : '[0-9x]{1,3}'
    const financialInstitutionsRx = new RegExp(` ${institutionCountRx} financial institutions`)
    
    cy.visit(`${baseURLToVisit}/data-browser/graphs/quarterly`)
    cy.wait(1000)
    cy.get('.heading > :nth-child(2)').contains(financialInstitutionsRx)
  })
  ```
   
  I've also rebased this branch onto `master` so that all the changes from the `direct-linking` branch are included.
   
  To pull down the updated branch you should be able to run `git pull --rebase`.